daliuge-engine/test/test_drop.py

```diff
@@ -1275,7 +1281,7 @@ def _test_multi_branch_graph(self, *results):
             last_first_output = y
 
         with DROPWaiterCtx(
-            self, all_drops, 2, [DROPStates.COMPLETED, DROPStates.SKIPPED]
+            self, all_drops, 300, [DROPStates.COMPLETED, DROPStates.SKIPPED]
```

Comment:
Any reason why this is 5 minutes long?

Reply:
To allow the debugging of the underlying code. But in reality it is just a left over from such a session.